content/applications/services/helpdesk/overview/getting_started.rst

```diff
@@ -2,63 +2,128 @@
 Getting Started
 ===============
 
-Helpdesk teams provide your customers with support to queries or errors they might encounter while
-using your product/service. Therefore, a successful scheme where you can organize multiple teams
-with their customized pipeline, visibilities settings, and ticket traceability is essential.
+Helpdesk teams provide customers with support to queries or issues they might encounter while using
+a product/service. Therefore, having a system in place where multiple teams can co-exist with their
+own customized pipeline, visibility settings, and ticket traceability is essential.
 
 Set up teams
 ============
 
-| To modify or create teams, go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`.
-| Setting up multiple teams allows you to group tickets by your channels (example: BE/US), or by
-  your support services' types (example: IT, accounting, admin, etc.).
+Setting up multiple teams allows for tickets to be grouped by location
+(example: :abbr:`BE/US(Belgium/United States)` ), or by support type (example: IT, accounting, etc.).
 
-.. image:: getting_started/helpdesk_teams_view.png
+To modify or create teams, go to :menuselection:`Helpdesk --> Configuration --> Teams`.
+
+.. image:: getting_started/helpdesk_teams_list.png
    :align: center
    :alt: View of the helpdesk teams page in Odoo Helpdesk
 
-Team’s productivity and visibility
-----------------------------------
+Assignments and visibility
+--------------------------
+
+Visibility
+~~~~~~~~~~
+The *Visibility* feature specifies who can view this team:
+
+- :guilabel:`Invited internal users`: Internal users will have access to the team and tickets
+  they are following. This access can be modified on each individual ticket.
+- :guilabel:`All internal users`: All internal users will be able to access the team and all of its
+  tickets without being a follower.
+- :guilabel:`Invited portal users and all internal users`: All internal users can access the team
+  and all of its tickets without being a follower. Portal users will only be able to access tickets
+  that they are following.
 
-Teams can have individual *Assignment Methods* to ensure that tickets get redirected to the right
-person:
+This guarantees that tickets with sensitive or proprietary information will only be seen by the
+appropriate users.
```

Comment:
What is `This`?

What is the most restrictive option out of the three listed here?
Can you give an `.. example::` ?